code/modules/client/loadout/loadout_hat.dm

```diff
@@ -44,4 +185,25 @@
 /datum/gear/hat/delinquent
 	display_name = "delinquent hat"
 	path = /obj/item/clothing/head/delinquent
-	cost = 5000
\ No newline at end of file
+	cost = 5000
+
+/datum/gear/hat/cueball
+	display_name = "cubeball helmet"
+	path = /obj/item/clothing/head/cueball
+	cost = 5000
+
+/datum/gear/hat/piratehat
+	display_name = "pirate hat"
+	description = "Yarr. Comes with one free pirate speak manual."
+	path = /obj/item/clothing/head/pirate
+	cost = 5000
+
+/datum/gear/hat/tinfoil
```

Comment:
I don't know if this should be an option, but I'm leaning towards no.

Reply:
agree